linux_os/guide/services/ntp/chronyd_run_as_chrony_user/tests/empty.fail.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+yum -y install ntp
+
+echo "" > /etc/sysconfig/ntpd
+echo "" > /usr/lib/systemd/system/ntpd.service
```

Comment:
What is the aim of this test?

Reply:
In ansible I am using back references while replacing values. And this option changes a bit behavior of lineinfile module. That is also the reason why there is one more task which is run in case the file is empty or missing. This test ensures it works properly in case of empty files.

Reply:
But it should install and make empty the chrony files, not ntpd file, right?

Reply:
oh right, fixed